sdcm/tester.py

```diff
@@ -559,14 +559,28 @@ def argus_get_scylla_version(self):
             severity=Severity.ERROR,
         )
 
+    def _is_test_error(self, events: dict[str, list[str]]) -> bool:
```

Comment:
`_is_test_error` is very broad and vague, when reading SCT code, one must peek inside what it means.
Let's call it `_cloud_resources_error`

Reply:
I think the confusion here is that @k0machi was trying to name it after the state change it's gonna do.
i.e. the state is called 'Test-Error"

the state is vague and board, hence the function might be.

we could name it differently, but once we'll introduce something else into the fold.
like some failures of KMS at boot, it won't be resources anymore, we'll need to refactor/rename it.

maybe `should_we_change_test_state_to_test_error` :)

Reply:
ok, but then let's have docstring explaining what we search for and update it when adding more checks

Reply:
Added docstring